_posts/2022-12-05-Introducing-Identity.markdown

```diff
@@ -0,0 +1,40 @@
+# Introducing Identity for OpenSearch
+
+OpenSearch handles critical data; how that data is permissioned and how a cluster is operated is a Big Deal™.  Consider the question "Who can delete all the indices in the cluster?", in the core of OpenSearch, the answer is whoever can make a request to the cluster.  
+
+Within the OpenSearch Project, the Security plugin allows finer levels of access control providing a more satisfying answer.  Security uses the plugin architecture of OpenSearch that forces concepts around identity to be optional in core, plugins, and extensions components.  Optional security is not enough for critical data, we will do to do better.

```

Comment:
It might be helpful to include a straightforward definition of identity here. What is it, and how does it change OpenSearch's approach to security?

Reply:
Thanks for the great feedback and the offline session - I think I've restructured the content and delivery much more cleanly and properly focused on introducing these concepts and their values, love to know more about what you think.